pelicun/model/damage_model.py

```diff
@@ -1296,6 +1304,74 @@ def map_ds(values: np.ndarray, offset: int) -> np.ndarray:
 
             return ds_id
 
+        def parse_scaling_specification(scaling_specification: dict) -> dict:  # noqa: C901
+            """
+            Parse and validate the scaling specification.
+
+            Parameters
+            ----------
+            scaling_specification : dict
+                The scaling specification to be parsed and validated.
+
+            Returns
+            -------
+            dict
+                The parsed and validated scaling specification.
+
+            Raises
+            ------
+            ValueError
+                If the scaling specification is invalid.
+            TypeError
+                If the type of an entry is invalid.
+            """
+            # if there are contents, ensure they are valid.
+            # See docstring for an example of what is expected.
+            parsed_scaling_specification = {}
+            # validate contents
+            for key, value in scaling_specification.items():
+                # loop through limit states
+                if key not in parsed_scaling_specification:
+                    parsed_scaling_specification[key] = {}
```

Comment:
There's no need to initialize keys in dictionaries like this. Use `defaultdict` instead. It is already used at a few places in pelicun to streamline such dictionary-building operations. For example, here: https://github.com/NHERI-SimCenter/pelicun/blob/ae8b35b7b4b63e5d0af2173eeda7f871145e4627/pelicun/model/demand_model.py#L1281

In this particular case, you can initialize the dict like this:
`parsed_scaling_specification = defaultdict(dict)`

and then the following code in line 1369 will just work:
`parsed_scaling_specification[key][LS] = []`